netmiko/nokia/nokia_srl.py

```diff
@@ -53,6 +53,32 @@ def session_preparation(self) -> None:
             self.disable_paging(command=command, cmd_verify=True, pattern=r"#")
         self.set_base_prompt()
 
+    def strip_prompt(self, *args: Any, **kwargs: Any) -> str:
+        """Strip the line1 of multiline prompt from the output."""
```

Comment:
is this statement correct about the 1st line? Shouldn't the strip tackle the two lines?

Reply:
Still calling super which I'd assume would strip the original prompt.

Reply:
I updated the docstrings, the `super()` call will get rid of `A:srl1#`.

The `_strip_context_items` call will get rid of things like `--{ running }...`

Reply:
ah, I see, by "prompt" you mean only one line of the SR Linux prompt.
Ok, lgtm